src/rust/ide/src/ide.rs

```diff
@@ -74,6 +74,12 @@ pub struct Ide {
 impl Ide {
     /// Constructor.
     pub async fn new(application:Application, view:ide_view::project::View, project:model::Project) -> FallibleResult<Self> {
+
+        #[cfg(debug_assertions)]
+        analytics::remote_log_data_field("debug_mode", "debug_mode_is_active", analytics::AnonymousData("true"));
+        #[cfg(not(debug_assertions))]
+        analytics::remote_log_data_field("debug_mode", "debug_mode_is_active", analytics::AnonymousData("false"));
+
```

Comment:
It this should be the first log, I probably put it in initialization function. Please note, that the view is already created and could already make some logs (however it shouldn't because there should be not any user interaction yet - still I would keep things consistent).

Reply:
Moved.